src/app/docs/kmcp/deploy/install-controller/page.mdx

```diff
@@ -29,7 +29,15 @@ The kmcp controller manages the lifecycle of MCP servers that are defined in an
    kind create cluster
    ```
 
-2. Install the following kmcp controller components in your cluster. 
+2. Install the kmcp CRDs.
+
+   ```sh
+   helm install kmcp-crds oci://ghcr.io/kagent-dev/kmcp/helm/kmcp-crds \
```

Comment:
I can't see it here, did you also add this instruction to the install page for `kagent` with helm?